src/simplewallet/simplewallet.cpp

```diff
@@ -1090,6 +1074,71 @@ bool simple_wallet::init(const boost::program_options::variables_map& vm)
       bool r = new_wallet(vm, m_recovery_key, m_restore_deterministic_wallet, m_non_deterministic, old_language);
       CHECK_AND_ASSERT_MES(r, false, tr("account creation failed"));
     }
+    if (!m_restore_height && m_restoring)
+    {
+      uint32_t version;
+      if (!try_connect_to_daemon(false, &version))
```

Comment:
This means you can't restore offline anymore. Maybe it should only do this if supplying a date rather than a block number.